src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -2405,6 +2409,13 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                 break;
             }
 
+            case NI_Sve_LoadVectorByteZeroExtendFirstFaulting:
+            case NI_Sve_LoadVectorSByteSignExtendFirstFaulting:
+            {
+                GetEmitter()->emitIns_R_R_R_R(ins, emitSize, targetReg, op1Reg, op2Reg, REG_ZR, opt);
```

Comment:
This needs restoring ffr using `wrffr` too as done on line 2400.

Reply:
Done.

Reply:
any reason why this cannot be combined with above cases? @SwapnilGaikwad  - can you please take care of it in your PR, since you will add more cases in it anyway.

Reply:
It's not possible to combine the two cases. This causes `assert(insScalableOptsNone(sopt))` to fail down in `emitInsSve_R_R_R_R()` for `INS_sve_ldff1b` at [line 6293](https://github.com/mikabl-arm/dotnet-runtime/blob/sve-loadvectorff/src/coreclr/jit/emitarm64sve.cpp#L6293). I've decided that it's easier to split the case blocks rather than implementing a clever fall-through. The value of `opt` here depends on the returned type (dest vector register elements' size).